corehq/toggles.py

```diff
@@ -282,6 +282,12 @@ def toggles_dict(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+LOCATION_TYPE_STOCK_RATES = StaticToggle(
+    'location_type_stock_rates',
+    "Specify stock rates per location type.",
```

Comment:
This convention seems like something to maybe chat about as a group, as you can see it's not used in (some/most?) neighboring instances of the toggle. I guess I don't mind it, it adds value sometimes for some people, but just looks funny for the rest of us.